src/components/Common/Navigation/Connectivity/ConnectivityWrap/ConnectivityWrap.js

```diff
@@ -53,7 +57,7 @@ export default function ConnectivityWrap({ children, serverStatus, origin }) {
         startReceiveInvHandler(
           origin,
           invitationId,
-          agent,
+          HOLDER_LABEL,
```

Comment:
@dc-andysign this fixes the bug you mentioned

Reply:
thanks:)